src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -407,10 +508,12 @@ export default class ManualBusinessInfo extends Mixins(CountriesProvincesMixin,
   private onComponentValid (): boolean {
     // this form is valid if we have the home jurisdiction (custom component)
     // and we have the home incorporation date (custom component)
+    // and we have the affidavit file, if required (custom component)
     // and the other form (Vuetify) components are valid
     return (
       !!this.business.homeJurisdiction &&
       !!this.business.homeIncorporationDate &&
+      (!this.isContinuationInAffidavitRequired || !!this.business.affidavitFileKey) &&
       this.formValid
     )
   }
```

Comment:
You also need to copy the styles for the bullets:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/29f88e9c-7a8d-4104-906e-a1f71703ce20)

Reply:
Done, updated